include/s3select.h

```diff
@@ -1218,6 +1226,29 @@ void push_between_filter::builder(s3select* self, const char* a, const char* b)
   self->getAction()->exprQ.push_back(func);
 }
 
+void push_not_between_filter::builder(s3select* self, const char* a, const char* b) const
+{
+  std::string token(a, b);
```

Comment:
looks like token is unused;  I also don't recognize this std::string constructor.  What are we trying to do here?

Reply:
Removed it. It was part of the previous commit